test/tests

```diff
@@ -86,28 +92,71 @@ test::composer::php80() {
         startSkipping
     fi
 
-    test::helpers::test_deploy "composer_php80" "PHP (composer.json)" "8.0."
+    test::utils::setupFixture "composer_php80"
+    test::helpers::test_deploy "PHP (composer.json)" "8.0."
 }
 
 test::composer::php81() {
     # Test a deployment of a PHP app using Composer
     # Specifying we want PHP 8.1.x in composer.json
 
-    test::helpers::test_deploy "composer_php81" "PHP (composer.json)" "8.1."
+    test::utils::setupFixture "composer_php81"
+    test::helpers::test_deploy "PHP (composer.json)" "8.1."
 }
 
 test::composer::php82() {
     # Test a deployment of a PHP app using Composer
     # Specifying we want PHP 8.2.x in composer.json
 
-    test::helpers::test_deploy "composer_php82" "PHP (composer.json)" "8.2."
+    test::utils::setupFixture "composer_php82"
+    test::helpers::test_deploy "PHP (composer.json)" "8.2."
 }
 
 test::composer::php83() {
     # Test a deployment of a PHP app using Composer
     # Specifying we want PHP 8.3.x in composer.json
 
-    test::helpers::test_deploy "composer_php83" "PHP (composer.json)" "8.3."
+    test::utils::setupFixture "composer_php83"
+    test::helpers::test_deploy "PHP (composer.json)" "8.3."
+}
+
+test::composer::over_defaults() {
```

Comment:
suggestion: I think these function names should include that we test the PHP version installed.
```suggestion
test::composer::php_version_defaults() {
```